tests/product/test_collect.py

```diff
@@ -147,24 +141,6 @@ def test_system_info_pa_separate_node(self):
             hosts=self.cluster.slaves)
 
     @attr('smoketest')
-    def test_collect_query_info(self):
```

Comment:
This can be removed because `test_query_info_pa_separate_node()` is basically the same test.